Upstreamized patch wrt http://bugs.gentoo.org/attachment.cgi?id=273059
Fixes race condition where two different xulrunners are installed, and
old headers get picked up by mistake.

--- src/plugin_setup.cpp-old	2009-06-09 19:48:33.518153163 -0500
+++ src/plugin_setup.cpp	2009-06-09 19:48:12.522163011 -0500
@@ -362,7 +362,7 @@
 
     }
 
-    nperror = NPN_GetValue (instance->mInstance, NPNVSupportsWindowless, &supportsWindowless);
+    nperror = NPN_GetValue (instance->mInstance, (NPNVariable) 17 /* NPNVSupportsWindowless */, &supportsWindowless);
 	supportsWindowless = (nperror == NPERR_NO_ERROR) && supportsWindowless;
 	if (instance->windowless) {
 		if (supportsWindowless) {

